### ISSUE TITLE
Does not work with files that have a .phtml extension

### ISSUE THREAD

**Issue by maxbrugnerotto** [vincaslt/vscode-highlight-matching-tag#80]
If I change the file extension to .html the visual studio code extension works very fine

**Comment by vincaslt** [comment]
It is strange, the plugin is independent of the file extension. Could you provide the content of the file?

**Comment by vincaslt** [comment]
Maybe some other plugin is interfering somehow. I did not get any zip file. You can create a gist: https://gist.github.com and share it.

**Comment by maxbrugnerotto** [comment]
[Screenshot_and_source_files.zip](https://github.com/vincaslt/vscode-highlight-matching-tag/files/3210894/Screenshot_and_source_files.zip)
Hi. I have attached the ZIP file. I have only 1 additional plugin : PHP Debug 1.13.0 (see image into attached ZIP file). Many thanks for your reply and best regards.

**Comment by vincaslt** [comment]
This is strange, it works with both files for me. Could you also share your plugin configuration (if you have any)?

**Comment by maxbrugnerotto** [comment]
Hi Vincaslt, many thanks for you reply.  I have only two extensions installed and enabled (see image below) :

1. Highlight Matching Tag   version 0.9.1
2. PHP Debug   version 1.13.0

Best regards
Massimiliano
![VisualStudioCode_Extensions](https://user-images.githubusercontent.com/50020033/58417522-51081280-8085-11e9-856a-5fb902851b34.png)

**Comment by maxbrugnerotto** [comment]
Hi Vincaslt, If you intend the file settings.json this should be the configuration file  :

**{
    "workbench.colorTheme": "Visual Studio Light",
    "editor.formatOnPaste": true,
    "php.validate.executablePath": "C:\\xampp\\php\\php.exe",
    "phpserver.phpPath": "C:\\xampp\\php\\php.exe",
    "editor.minimap.enabled": true,
    "highlight-matching-tag.highlightFromContent": true,
    "highlight-matching-tag.highlightSelfClosing": true,
    "editor.detectIndentation": false
}**

**Comment by maxbrugnerotto** [comment]
This is a screen-shot of settings : 
![VisualStudioCode_Highlight_settings](https://user-images.githubusercontent.com/50020033/58418029-e1932280-8086-11e9-86aa-06b54b3db7c3.png)

**Comment by kanlukasz** [comment]
> Hi Vincaslt, If you intend the file settings.json this should be the configuration file :
> 
> "php.validate.executablePath": "C:\xampp\php\php.exe",
> "phpserver.phpPath": "C:\xampp\php\php.exe"

If you use Windows, path should have two `\\`, for example : `"C:\\xampp\\php\\php.exe"`

**Comment by vincaslt** [comment]
I am still getting the expected result on windows, and can't imagine why it would work that way for you... I managed to reproduce the issue only once on OSX, but can't reproduce it anymore.

**Comment by meelos** [comment]
Any way to enable this extension on html.erb files? :)

**Comment by vincaslt** [comment]
It should be enabled by default on any file. It might be crashing because of some unexpected syntax, in which case I advise you to register a separate issue with the code example which doesn't work.
